### ISSUE TITLE
[Quest][Classic] Red Snapper - Very Tasty Bugged Mobs

### ISSUE THREAD

**Issue by Allelya** [dalaranwow/dalaran-wow#4001]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Quest "Red Snapper Very Tasty" has bugged elements. When using the quest item fishing net on the school of red snapper, there is a chance for an Angry Murloc mob to spawn. Upon attacking the Angry Murloc, it will either despawn at half health or despawn at low health when fleeing.

**Expected behaviour**: Angry Murloc mob that spawns when using the fishing net should NOT DESPAWN and should instead die and give the quest item loot.

**Steps to reproduce the problem**:

1. Pick up the quest Red Snapper Very Tasty.
2. Go to the school of red snapper and use the fishing net
3. Attack a spawned Angry Murloc 

**Include proofs for this behaviour**
http://www.wowhead.com/quest=9452/red-snapper-very-tasty
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
NPC- Diktynna, Quest Item Fishing Net- inv_misc_net_01, Angry Murloc- no ID
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Quest: Red Snapper - Very Tasty - Item is no longer usable far away from the schools + Angry Murloc will no longer despawn after 15 seconds

* closes https://github.com/dalaranwow/dalaran-wow/issues/4001
* commit: e8ca06554201096e74174e24b7980a0f2c7711a6

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
